docs/other/troubleshooting-gratia.md

```diff
@@ -260,7 +261,12 @@ Not much is printed to the screen, but you can see progress in the Gratia log fi
 
 
 !!! note 
-    Condor rotates history files, so you can only report what Condor has kept. Controlling the Condor history is documented in the Condor manual. In particular, see the options for [MAX_HISTORY_LOG](https://research.cs.wisc.edu/htcondor/manual/v8.6/3_5Configuration_Macros.html#21780) and [MAX_HISTORY_ROTATIONS](https://research.cs.wisc.edu/htcondor/manual/v8.6/3_5Configuration_Macros.html#21785).
+    Condor rotates history files, so you can only report what Condor has kept.
```

Comment:
```suggestion
    HTCondor rotates history files, so you can only report what Condor has kept.
```

Reply:
I dunno, there are a lot of references to "Condor" (the batch system) throughout the document, including two in this sentence.  Do you want to convert them all?

Reply:
Yeah, let's strive to use the official name for software in our docs. I would imagine that non-Emacs editors can handle a find-and-replace :)

Reply:
Yes although it's slightly more nuanced because sometimes Condor refers to HTCondor and sometimes Condor refers to the condor probe (ie, gratia-probe-condor).

Reply:
Ah, that's very confusing! In the latter case, let's make it clearer with something like `the HTCondor Gratia probe` or `gratia-probe-condor`

Reply:
Alrighty